flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkScanSql.java

```diff
@@ -52,12 +53,25 @@ public TestFlinkScanSql(String fileFormat) {
   @Override
   public void before() throws IOException {
     super.before();
-    tEnv = TableEnvironment.create(EnvironmentSettings.newInstance().useBlinkPlanner().inBatchMode().build());
-    tEnv.executeSql(String.format(
+    getTableEnv().executeSql(String.format(
```

Comment:
Why do we need to change here ?

Reply:
this is to avoid creating the table environment for each test method. it is the same pattern used by `FlinkTestBase`.

Reply:
OK, that sounds good to me.   

Nit: here we could just use the `executeSQL` methods now.   Maybe we could just rename the `executeSQL(String sql)` as `sql(String query, Object... args)`, so that we could have the same usage that is similar to the [FlinkTestBase#sql](https://github.com/apache/iceberg/blob/543a6cdd0538e505c4ab3fa0f0876a5a22bd7704/flink/src/test/java/org/apache/iceberg/flink/FlinkTestBase.java#L88).

Reply:
really like the suggestion. will update